### PR TITLE
Better linters

### PR DESCRIPTION
**Pull request by smiller171**

### CODE REVIEW

File: .github/workflows/pullRequest.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+name: pull request
```
Comment (smiller171) [inline .github/workflows/pullRequest.yaml:2]:
```suggestion
---
name: pull request
```

File: .github/workflows/push.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
+name: push
```
Comment (smiller171) [inline .github/workflows/push.yaml]:
```suggestion
---
name: push
```

File: lambda/.eslintrc.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+env:
```
Comment (smiller171) [inline lambda/.eslintrc.yml:2]:
```suggestion
---
env:
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# terraform-anycloud-template [![](https://github.com/rhythmictech/terraform-anycloud-template/workflows/pre-commit-check/badge.svg)](https://github.com/rhythmictech/terraform-anycloud-template/actions) <a href="https://twitter.com/intent/follow?screen_name=RhythmicTech"><img src="https://img.shields.io/twitter/follow/RhythmicTech?style=social&logo=RhythmicTech" alt="follow on Twitter"></a>
+Template repository for terraform modules. Good for any cloud and any provider.
```
Comment (sblack4) [inline README.md]: This needs updating

File: variables.tf [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# variable "name" {
+#   description = "Moniker to apply to all resources in the module"
+#   type        = string
+# }
+
+# variable "tags" {
+#   default     = {}
+#   description = "User-Defined tags"
+#   type        = map(string)
+# }
```
Comment (sblack4) [inline variables.tf:10]: delete?

**Review by sblack4 — CHANGES_REQUESTED** [review]
The readme needs some love

File: variables.tf (hunk 5 above)
Comment (smiller171) [reply variables.tf:10]: I'm gonna be working with those on my next PR so I didn't want them totally gone, just not throwing the "declared but not used" error

File: README.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+# lambda-update-image-builder-pipeline
+Template repository for terraform modules. Good for any cloud and any provider.
```
Comment (sblack4) [inline README.md:2]: I guess update the subtitle and stuff in a subsequent PR?

**Review by sblack4 — APPROVED** [review]
:shipit:

File: README.md (hunk 6 above)
Comment (smiller171) [reply README.md:2]: ugh...yeah I'll update it in the one that actually adds the terraform stuff. Gonna rename the repo too. Originally it was gonna be 2 separate repos, but I like it better this way

**Pull request merged**